### PR TITLE
Digitigrade legs port

### PR DESCRIPTION
**Pull request by KillianKirilenko**
Upstreaming digitigrade legs from chomp, finally.
image

I apologize in advance to downstream for any issues this causes.

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -1059,3 +1062,61 @@
...
+		var/mob/living/carbon/human/H = user
+
+		// if digitigrade-use flag is set
+		if(H.digitigrade)
+
+			// figure out what slot we care about
+			var/update_icon_define_digi = null
+
+			if(istype(src, /obj/item/clothing/shoes))
+				update_icon_define_digi = "icons/inventory/feet/mob_digi.dmi"
```
Comment (Heroman3003) [inline code/modules/clothing/clothing.dm:1077]: Ideally, that could be a variable on the clothing obj itself, rather than determined in the proc tbh.

Comment (KillianKirilenko) [reply code/modules/clothing/clothing.dm:1077]: It does strike me as odd that they didn't do it that way originally now that I look at it, but unfortunately untangling it myself might take a while. I'm an amateurish coder at best.

Comment (Heroman3003) [reply code/modules/clothing/clothing.dm:1077]: Well that specific part shouldn't be hard to fix, I imagine, so I'll leave it up for a bit.

Comment (KillianKirilenko) [reply code/modules/clothing/clothing.dm]: Took a shot at it and it seems to compile and run with no runtimes, and changing clothes between planti/digi characters has no issues in my local server either.

**Review by Heroman3003 — APPROVED** [review]

**Pull request merged**